### PR TITLE
add ex 7 solution

### PR DESCRIPTION
**Pull request by stevelee2497**
get json file from url and convert it to xml file

### CODE REVIEW

File: exercise7/Program.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            var doc = JsonConvert.DeserializeXmlNode(response.Content, "root");
+            var fileName = GetXmlFileName(url);
+            File.WriteAllText(fileName, XDocument.Parse(doc.InnerXml).ToString());
+        }
+
+        private static string GetXmlFileName(string url)
+        {
+            var splittedUrl = url.Split('/');
+            var fileName = splittedUrl[splittedUrl.Length - 1].Replace("json", "xml");
+            return fileName;
```
Comment (voquanghoa) [inline exercise7/Program.cs]: Easier `return url.Substring(url.LastIndexOf("/")+1).Replace("json", "xml");`

File: exercise7/Program.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                "http://danabus.vn/data/2/bus-services.json",
+                "http://danabus.vn/data/2/bus-services/5.json",
+                "http://danabus.vn/data/2/bus-services/7.json",
+                "http://danabus.vn/data/2/bus-services/8.json",
+                "http://danabus.vn/data/2/bus-services/11.json",
+                "http://danabus.vn/data/2/bus-services/12.json"
+            };
+            foreach (var url in urls)
+            {
+                ConvertJsonToXml(url);
```
Comment (voquanghoa) [inline exercise7/Program.cs]: Declare `urls` as a list then you can call `urls.ForEach(ConvertJsonToXml)`

Comment (stevelee2497) [reply exercise7/Program.cs]: done

File: exercise7/Program.cs (hunk 1 above)
Comment (stevelee2497) [reply exercise7/Program.cs]: done

**Pull request merged**